test/e2e/generator/generate.go

```diff
@@ -43,13 +42,7 @@ var (
 		"kill":       0.1,
 		"restart":    0.1,
 	}
-	nodeMisbehaviors = weightedChoice{
-		// FIXME: evidence disabled due to node panicing when not
-		// having sufficient block history to process evidence.
-		// https://github.com/tendermint/tendermint/issues/5617
-		// misbehaviorOption{"double-prevote"}: 1,
-		misbehaviorOption{}: 9,
-	}
+	evidence = uniformChoice{0, 1, 10}
```

Comment:
What are these choices between?

Reply:
no evidence, 1 evidence of 10 pieces of evidence